crates/rust-analyzer/src/config.rs

```diff
@@ -244,6 +244,9 @@ config_data! {
 
         /// Whether to show documentation on hover.
         hover_documentation_enable: bool       = "true",
+        /// Whether to show keyword hover popups. Only applies when
+        /// `#rust-analyzer.hover.documentation.enable#` is set.
+        hover_documentation_keywords: bool     = "true",
```

Comment:
```suggestion
        hover_documentation_keywords_enable: bool     = "true",
```
This should probably have the (possibly redundant) `enable` suffix to be consistent with the rest of the configs